File-Formats/XlsIO/Working-with-Template-Markers.md

```diff
@@ -648,51 +648,51 @@ The output of all the image insertion options with input templates are as follow
 
 Input template
 
-<img src="Working-with-Template-Markers_images/Working-with-Template_Markers_img6.jpeg" alt="Import collection objects with images Example" width="100%" Height="Auto"/>
+<img src="working-with-template-markers_images/file-formats-xlsio-imput-template-default.jpeg" alt="Imput Template Default in XLSIO " width="100%" Height="Auto"/>
 
 Generated output
 
-<img src="Working-with-Template-Markers_images/Working-with-Template_Markers_img7.jpeg" alt="Import collection objects with images Example" width="100%" Height="Auto"/>
+<img src="working-with-template-markers_images/file-formats-xlsio-general-output-default.jpeg" alt="General Output Default in XLSIO" width="100%" Height="Auto"/>
 
 **Image** **with** **FitToCell** **attribute**
 
 Input template
 
-<img src="Working-with-Template-Markers_images/Working-with-Template_Markers_img8.jpeg" alt="Import collection objects with images Example" width="100%" Height="Auto"/>
+<img src="working-with-template-markers_images/file-formats-xlsio-imput-template-attribute.jpeg" alt="Imput Template Attribute in XLSIO" width="100%" Height="Auto"/>
 
 Generated output
 
-<img src="Working-with-Template-Markers_images/Working-with-Template_Markers_img9.jpeg" alt="Import collection objects with images Example" width="100%" Height="Auto"/>
+<img src="working-with-template-markers_images/file-formats-xlsio-general-output-attribute.jpeg" alt="General Output Attribute in XLSIO" width="100%" Height="Auto"/>
 
 **Image** **with** **Size**
 
 Input template
 
-<img src="Working-with-Template-Markers_images/Working-with-Template_Markers_img10.jpeg" alt="Import collection objects with images Example" width="100%" Height="Auto"/>
+<img src="working-with-template-markers_images/file-formats-xlsio-imput-template-size.jpeg" alt="Imput Template Size in XLSIO" width="100%" Height="Auto"/>
 
 Generated output
 
-<img src="Working-with-Template-Markers_images/Working-with-Template_Markers_img11.jpeg" alt="Import collection objects with images Example" width="100%" Height="Auto"/>
+<img src="working-with-template-markers_images/file-formats-xlsio-general-output-Size.jpeg" alt="General Output Size in XLSIO" width="100%" Height="Auto"/>
```

Comment:
Why you using upper case? Check the image name.

Reply:
Modified correctly